client/rest-high-level/src/main/java/org/elasticsearch/client/ml/dataframe/Classification.java

```diff
@@ -92,13 +102,15 @@ public static Builder builder(String dependentVariable) {
     private final Integer numTopFeatureImportanceValues;
     private final String predictionFieldName;
     private final Double trainingPercent;
+    private final ClassAssignmentObjective classAssignmentObjective;
     private final Integer numTopClasses;
     private final Long randomizeSeed;
 
     private Classification(String dependentVariable, @Nullable Double lambda, @Nullable Double gamma, @Nullable Double eta,
                            @Nullable Integer maximumNumberTrees, @Nullable Double featureBagFraction,
                            @Nullable Integer numTopFeatureImportanceValues, @Nullable String predictionFieldName,
-                           @Nullable Double trainingPercent, @Nullable Integer numTopClasses, @Nullable Long randomizeSeed) {
+                           @Nullable Double trainingPercent, @Nullable ClassAssignmentObjective classAssignmentObjective,
+                           @Nullable Integer numTopClasses, @Nullable Long randomizeSeed) {
```

Comment:
Is there a reason for not adding it as the last param of the constructor (and possibly the members list above too)? If not, I think it'd be best to move it last as it would make it simpler for users of the client to address upon upgrading.

Reply:
No good point.

Reply:
[4e0ded7](https://github.com/elastic/elasticsearch/pull/52763/commits/4e0ded77b33c9699553bc1a6e5df1216433fc3a0)